src/app/submission/objects/submission-objects.reducer.ts

```diff
@@ -856,3 +860,20 @@ function deleteFile(state: SubmissionObjectState, action: DeleteUploadedFileActi
   }
   return state;
 }
+
+function cleanDuplicateDetectionSection(state: SubmissionObjectState, action: CleanDuplicateDetectionAction): SubmissionObjectState {
+  if (isNotEmpty(state[ action.payload.submissionId ])) {
+    return Object.assign({}, state, {
+      [ action.payload.submissionId ]: Object.assign({}, state[ action.payload.submissionId ], {
+        sections: Object.assign({}, state[ action.payload.submissionId ].sections, {
+          [ 'duplicates' ]: Object.assign({}, state[ action.payload.submissionId ].sections.duplicates, {
```

Comment:
Is this maybe where the error occurs?  It appears this might be running even if duplicate detection is disabled.

Reply:
I found an error that occurs in submission-objects effects (and potentially in reducer as well as you identified), and added fixes so that there is no assumption the 'duplicates' section is present when it tries to clean/remove it.

Regarding the "no duplicates link" - this is similar to the error i saw with profiles, and which the e2e tests in this PR encountered once.
Regarding duplicates, this is sort of a problem we introduced - when we use that annotation to disable the controller when the feature is disabled, it is no longer registered in teh main list of endpoints. That is where the 'duplicates' is missing from, when the data service tries to resolve the endpoint. That said, I'm not sure how or why your prod instance is trying to resolve that endpoint since that only occurs with a findDuplicates call, which itself is conditional on the config property being enabled.

Is it possible you changed the config property so the feature detection came back as 'enabled', but since tomcat wasn't fully restarted, the controller had not re-initialised and therefore was still 404'd / not registered in the endpoint lists? If so, is that a scenario we should anticipate? I could introduce a second config check into the data service itself, perhaps

Reply:
@kshepherd : I'm able to reproduce this issue with the latest code.  I restarted Tomcat completely with this feature disabled by default.  When I run the UI via `yarn build:prod; yarn serve:ssr`, I can reproduce this issue whenever I visit MyDSpace and select "Workflow tasks" from the dropdown.

I *know* other features are doing this properly...as we have a lot of features which only get enabled via an `enable` boolean.  I'm not exactly sure how they fix this issue, but there must be a workaround.  I'm looking around at other features for examples.

Reply:
I will try as well. Perhaps it is something as simple as forcing an initial state of 'false' for the config property if that observable somehow is initially evaluated in a state where the endpoint is attempted to be queried. (or if my combineLatest block is otherwise not quite perfect - async angular operations like this are not my area of expertise)

Reply:
I have to leave for the day...but I admit this isn't my area of expertise either.  I just know that we *don't* have these errors appearing all over the application every time we disable a feature on the backend with `@ConditionalOnProperty`. 

So, there has to be a way to achieve this, but I'm not sure myself how others have done this.  If it comes down to it, I can ask others in tomorrow's DevMtg to provide advice here.....just let me know if you cannot find a better solution.

(I'm *hoping* maybe if you can reproduce this it might become more obvious to you what may be happening...we'll see)